src/EditorFeatures/Test2/Rename/RenameTestHelpers.vb

```diff
@@ -97,6 +113,16 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
             Next
         End Function
 
+        Public Sub VerifyFileRename(workspace As TestWorkspace, newIdentifierName As String)
```

Comment:
Are there any cases where you're verifying that the file was _not_ renamed? Like when the session gets cancelled?

Reply:
Adding file name check to all tests, even if it's not expected to change. There are cancellation tests, so it will get covered